src/main/connector/system/Windows/Windows.yaml

```diff
@@ -128,7 +128,7 @@ monitors:
         fileSystemInformation:
           type: wmi
           namespace: root\CIMv2
-          query: SELECT DeviceID,FreeSpace,Size,VolumeName FROM Win32_LogicalDisk
+          query: SELECT DeviceID,FreeSpace,Size,VolumeName FROM Win32_LogicalDisk WHERE NOT DriveType = 5

```

Comment:
There are probably other drive types we want to exclude, notably shares, etc.

Reply:
Actually, we likely want to include only DriveType=3. The rest are as follows:

DriveType = 0: Unknown drive type
DriveType = 1: No root directory (invalid drive)
DriveType = 2: Removable drive (e.g., floppy disk drive, USB flash drive)
DriveType = 3: Local disk drive (e.g., hard drive)
DriveType = 4: Network drive
DriveType = 5: CD-ROM drive
DriveType = 6: RAM disk